sklearn/linear_model/perceptron.py

```diff
@@ -114,7 +114,20 @@ class Perceptron(BaseSGDClassifier):
     n_iter_ : int
         The actual number of iterations to reach the stopping criterion.
         For multiclass fits, it is the maximum over every binary fit.
-
+      
+    Example
+    --------
+    >>> from sklearn.datasets import load_digits
+    >>> from sklearn.linear_model import perceptron
+    >>> from sklearn.model_selection import train_test_split
```

Comment:
We usually wouldn't do a train-test split in an example

Reply:
Thanks a lot @jnothman !